### ISSUE TITLE
How do I get back to MIDI file after nom_midi::parser::parse_midi ?

### ISSUE THREAD

**Issue by vi** [derekdreery/nom-midi-rs#1]
There should example of "round tripping" a MIDI file, maybe also while editing the notes (e.g. transpose each note).

**Comment by richard-uk1** [comment]
Yes there's no code generation in this lib yet - it should be easier to implement than parsing.
